src/components/MainComponent/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useState } from "react";
+import { createUserWithEmailAndPassword, signInWithEmailAndPassword } from "firebase/auth"
+import { auth } from "../../../Firebase/firebase";
+import "./LoginPage.scss"
+import Nutrition from "../Nutrition/Nutrition";
+import {Link} from "react-router-dom";
+
+const LoginPage = ({ type, label }) => {
+    const [userName, setUserName] = useState("")
+    const [password, setPassword] = useState("")
+    const [error, setError] = useState("")
+    const register = () => {
+        createUserWithEmailAndPassword(auth, userName, password)
+            .then(user => {
+                setError("")
+                setUserName("")
+                setPassword("")
+            })
+            .catch(error => {
+                setError(error)
+            })
+    }
+
+    const login = () => {
+        signInWithEmailAndPassword(auth, userName, password)
+        .then(user => {
+                setError("")
```

Comment:
You didn't store user statusanywhere it will prevent you to know the status after this component will be unmounted